weather_dl/weather-dl

```diff
@@ -49,28 +45,4 @@ if __name__ == '__main__':
         except ImportError as e:
             raise ImportError('please re-install package in a clean python environment.') from e
 
-    if '-h' in sys.argv or '--help' in sys.argv or len(sys.argv) == 1:
-        cli()
-    else:
-        with tempfile.TemporaryDirectory() as tmpdir:
-            original_dir = os.getcwd()
-
-            # Convert subpackage to a tarball
-            os.chdir(site_pkg)
-            subprocess.check_call(
-                f'{sys.executable} ./setup.py -q sdist --dist-dir {tmpdir}'.split(),
-            )
-
-            os.chdir(original_dir)
-
-            # Set tarball as extra packages for Beam.
-            pkg_archive = glob.glob(os.path.join(tmpdir, '*.tar.gz'))[0]
-
-            with tarfile.open(pkg_archive, 'r') as tar:
-                assert any([f.endswith('.py') for f in tar.getnames()]), 'extra_package must include python files!'
-
-            # cleanup memory to prevent pickling error.
-            tar = None
-            weather_dl = None
-
-            cli(['--extra_package', pkg_archive])
+    cli()
```

Comment:
As mentioned above, I think the SDK container argument should be input here as a few list arguments in the CLI.

Reply:
Updated.